cmd/plugin/rpaasv2/cmd/scale.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2019 tsuru authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	"github.com/tsuru/rpaas-operator/cmd/plugin/rpaasv2/proxy"
+)
+
+type scaleArgs struct {
+	service  string
+	instance string
+	quantity int
+	prox     *proxy.Proxy
+}
+
+// scaleCmd represents the scale command
+var scaleCmd = &cobra.Command{
+	Use:   "scale",
+	Short: "A brief description of your command",
+	Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command. For example:
+
+Cobra is a CLI library for Go that empowers applications.
+This application is a tool to generate the needed files
+to quickly create a Cobra application.`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.ParseFlags(args)
+		scale := scaleArgs{}
```

Comment:
Nitpicking here... You could create this object directly instead of many field assignments.